### ISSUE TITLE
Two new braille tables in simplified Chinese.

### ISSUE THREAD

**Issue by vgjh2005** [nvaccess/nvda#5553]
Hi:
I have uploaded two braille tables in simplified Chinese to the SVN server. Please add them to the NVDA. The zh-cn-1.ctb is Chinese (China, mandarin braille);
The zh-cn-3.ctb is Chinese (China, double concatenation braille).
It's made by allan. Thanks a lot!

**Comment by jcsteh** [comment]
These should be submitted to the [liblouis](https://github.com/liblouis/liblouis) project. Once they're included in liblouis, they can be included soon thereafter by NVDA.

**Comment by josephsl** [comment]
Hi,

LibLouis 3 release notes say this has been incorporated into tables collection. Will be combined with #6109. Thanks.

**Comment by vgjh2005** [comment]
OK. Thanks josephs.

**Comment by ehollig** [comment]
Closing this, as #6109 is now implemented and https://github.com/liblouis/liblouis/issues/137 is closed.

**Comment by jcsteh** [comment]
Although we've updated liblouis in NVDA, there is no Chinese (China) table in NVDA's table list yet. Also, the tables mentioned here (zh-cn-1.ctb and zh-cn-3.ctb) are not in liblouis. Instead, there is a zh-chn.ctb. We should probably add this zh-chn.ctb. I have a few questions:

1. Should this be called Chinese (China), Chinese (China, Manderin) or Chinese (China, Simplified)?
2. @vgjh2005, do you know anything about this table? Do you still plan to submit the other tables to liblouis? What's different about them?

**Comment by vgjh2005** [comment]
Hi:
Sorry. I just saw this comment. I have downloaded this file to my computer from liblouis. But this file is useless. This file is made by pinyin simply. But the same character may have different pronounciation. So we made a largeest list to cover them. This issue has gone for long time. Braille configuration of simplified Chinese have been updated more times. So I suggest to replace that file. If it is needed, I will upload two braille configuration here again. Thanks!

**Comment by jcsteh** [comment]
You need to submit new tables to liblouis, not NVDA. Once they're included in liblouis, we can add them to NVDA's table list. Thanks.

**Comment by vgjh2005** [comment]
Hi:
I don't know how to do. Please tell me. Thanks a lot!

2017-09-04 6:02 GMT+08:00, James Teh <notifications@github.com>:

**Comment by ehollig** [comment]
@vgjh2005, please create a [New Issue at liblouis/liblouisutdml](https://github.com/liblouis/liblouisutdml/issues/new) with the tables attached.

**Comment by leonardder** [comment]
@vgjh2005: Are the tables mentioned in https://github.com/liblouis/liblouis/pull/159 the tables you'd like to see in nVDA?

**Comment by leonardder** [comment]
Looks like it according to https://github.com/liblouis/liblouis/pull/612#issuecomment-414040214
